### PR TITLE
synthetic monitors doc

### PR DESCRIPTION
**Pull request by sandeepkumar-kagitha**
# Description

Please include a summary of the change and which issue is fixed (if relevant).

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

Please delete options that are not relevant.

- [ ] My commit message follows [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [ ] My code is formatted to [Go standards](https://go.dev/blog/gofmt)
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes. Go [here](https://github.com/newrelic/terraform-provider-newrelic#testing) for instructions on running tests locally.

## How to test this change?

Please describe how to test your changes. Include any relevant steps in the UI, HCL file(s), commands, etc

- Step 1
- Step 2
- etc

### CODE REVIEW

File: website/docs/r/synthetics_monitor.html.markdown [hunk 1]
```diff
@@ -3,114 +3,115 @@ layout: "newrelic"
 page_title: "New Relic: newrelic_synthetics_monitor"
 sidebar_current: "docs-newrelic-resource-synthetics-monitor"
 description: |-
-  Create and manage a Synthetics monitor in New Relic.
+Create and manage a Synthetics monitor in New Relic.
 ---
 
 # Resource: newrelic\_synthetics\_monitor
 
-Use this resource to create, update, and delete a synthetics monitor in New Relic.
```
Comment (mbazhlekova) [inline website/docs/r/synthetics_monitor.html.markdown:11]: Use this resource to create, update, and delete a Simple or Browser Synthetics Monitor in New Relic.

File: website/docs/r/synthetics_monitor.html.markdown [hunk 2]
```diff
@@ -3,114 +3,115 @@ layout: "newrelic"
...
-The following arguments are supported:
+The following are the common arguments supported for `SIMPLE` and `BROWSER` monitors:
 
-  * `name` - (Required) The title of this monitor.
-  * `type` - (Required) The monitor type. Valid values are `SIMPLE`, `BROWSER`, `SCRIPT_BROWSER`, and `SCRIPT_API`.
-  * `frequency` - (Required) The interval (in minutes) at which this monitor should run.
-  * `status` - (Required) The monitor status (i.e. `ENABLED`, `MUTED`, `DISABLED`).
-  * `locations` - (Required) The locations in which this monitor should be run.
-  * `sla_threshold` - (Optional) The base threshold (in seconds) to calculate the [Apdex score](https://docs.newrelic.com/docs/apm/new-relic-apm/apdex/apdex-measure-user-satisfaction/) for use in the [SLA report](https://docs.newrelic.com/docs/synthetics/synthetic-monitoring/pages/synthetic-monitoring-aggregate-monitor-metrics/#viewing). Default is 7 seconds.
+* `validation_string` - (Optional) TValidation text for monitor to search for at given URI.
```
Comment (mbazhlekova) [inline website/docs/r/synthetics_monitor.html.markdown:68]:
```suggestion
* `validation_string` - (Optional) Validation text for monitor to search for at given URI.
```

File: website/docs/r/synthetics_monitor.html.markdown [hunk 3]
```diff
@@ -3,114 +3,115 @@ layout: "newrelic"
...
-
-```hcl
-resource "newrelic_synthetics_monitor" "foo" {
-  name = "foo"
-  type = "SCRIPT_BROWSER"
-  frequency = 5
-  status = "ENABLED"
-  locations = ["AWS_US_EAST_1"]
-}
+Warning: This resource will use the account ID linked to your API key. At the moment it is not possible to dynamically set the account ID.
```
Comment (mbazhlekova) [inline website/docs/r/synthetics_monitor.html.markdown:102]: This was only true for the REST API. We can remove it.

File: website/docs/r/synthetics_monitor.html.markdown (hunk 2 above)
Comment (mbazhlekova) [inline website/docs/r/synthetics_monitor.html.markdown:68]: We also need to add `account_id` attribute

File: website/docs/r/synthetics_monitor.html.markdown [hunk 4]
```diff
@@ -3,114 +3,115 @@ layout: "newrelic"
...
 
-  * `name` - (Required) The title of this monitor.
-  * `type` - (Required) The monitor type. Valid values are `SIMPLE`, `BROWSER`, `SCRIPT_BROWSER`, and `SCRIPT_API`.
-  * `frequency` - (Required) The interval (in minutes) at which this monitor should run.
-  * `status` - (Required) The monitor status (i.e. `ENABLED`, `MUTED`, `DISABLED`).
-  * `locations` - (Required) The locations in which this monitor should be run.
-  * `sla_threshold` - (Optional) The base threshold (in seconds) to calculate the [Apdex score](https://docs.newrelic.com/docs/apm/new-relic-apm/apdex/apdex-measure-user-satisfaction/) for use in the [SLA report](https://docs.newrelic.com/docs/synthetics/synthetic-monitoring/pages/synthetic-monitoring-aggregate-monitor-metrics/#viewing). Default is 7 seconds.
+* `validation_string` - (Optional) TValidation text for monitor to search for at given URI.
+* `verify_ssl` - (Optional) Monitor should validate SSL certificate chain.
+* `period` - (Required) The interval at which the monitor runs in minutes.
```
Comment (mbazhlekova) [inline website/docs/r/synthetics_monitor.html.markdown:70]: We should add the valid values for period.

File: website/docs/r/synthetics_monitor.html.markdown [hunk 5]
```diff
@@ -3,114 +3,115 @@ layout: "newrelic"
...
+* `status` - (Required) The run state of the monitor.
+* `locations_public` - (Required) The locations the monitor will run from.
```
Comment (mbazhlekova) [inline website/docs/r/synthetics_monitor.html.markdown:72]:
```suggestion
* `location_public` - (Required) The locations the monitor will run from.
```

File: website/docs/r/synthetics_monitor.html.markdown [hunk 6]
```diff
@@ -3,114 +3,115 @@ layout: "newrelic"
...
+* `name` - (Required) The human-readable identifier for the monitor.
```
Comment (mbazhlekova) [inline website/docs/r/synthetics_monitor.html.markdown:73]: Also need `location_private` attribute

File: website/docs/r/synthetics_script_monitor.html.markdown [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+layout: "newrelic"
+page_title: "New Relic: newrelic_synthetics_script_monitor"
+sidebar_current: "docs-newrelic-resource-synthetics-script-monitor"
+description: |-
+Create and manage a Synthetics script monitor in New Relic.
+---
+
+# Resource: newrelic\_synthetics\_script\_monitor
+
+Use this resource to create and manage New Relic synthetics script monitor.
```
Comment (mbazhlekova) [inline website/docs/r/synthetics_script_monitor.html.markdown:11]: Use this resource to create, update, and delete a Script API or Script Browser Synthetics Monitor in New Relic.

File: website/docs/r/synthetics_script_monitor.html.markdown [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+     }
+```
+
+## Argument Reference
+
+The following are the common arguments supported for `SCRIPT_API` and `SCRIPT_BROWSER` monitors:
+
+* `name` - (Required) The name for the monitor.
+* `type` - (Required) The plaintext representing the monitor script.
+* `locations_public` - (Required) The locations the monitor will run from.
```
Comment (mbazhlekova) [inline website/docs/r/synthetics_script_monitor.html.markdown:59]: Needs `location_private` attribute

File: website/docs/r/synthetics_script_monitor.html.markdown [hunk 9]
```diff
@@ -0,0 +1,96 @@
...
+		}
+     }
+```
+
+## Argument Reference
+
+The following are the common arguments supported for `SCRIPT_API` and `SCRIPT_BROWSER` monitors:
+
+* `name` - (Required) The name for the monitor.
+* `type` - (Required) The plaintext representing the monitor script.
```
Comment (mbazhlekova) [inline website/docs/r/synthetics_script_monitor.html.markdown:58]: We should add the valid `type` values - `Valid values are SCRIPT_BROWSER, and SCRIPT_API`

**Review by mbazhlekova — CHANGES_REQUESTED** [review]

File: website/docs/r/synthetics_monitor.html.markdown [hunk 10]
```diff
@@ -3,114 +3,115 @@ layout: "newrelic"
...
-  Create and manage a Synthetics monitor in New Relic.
+Create and manage a Synthetics monitor in New Relic.
 ---
 
 # Resource: newrelic\_synthetics\_monitor
 
-Use this resource to create, update, and delete a synthetics monitor in New Relic.
+Use this resource to create and manage New Relic Synthetics monitor.
 
 ## Example Usage
```
Comment (mbazhlekova) [inline website/docs/r/synthetics_monitor.html.markdown:13]: Can we also add one example with a private location? Using the private location resource to create one and then link to the monitor.

Comment (sandeepkumar-kagitha) [reply website/docs/r/synthetics_monitor.html.markdown:13]: Sure..Will add another example as well

File: website/docs/r/synthetics_script_monitor.html.markdown (hunk 7 above)
Comment (sandeepkumar-kagitha) [reply website/docs/r/synthetics_script_monitor.html.markdown]: Done

File: website/docs/r/synthetics_monitor.html.markdown [hunk 11]
```diff
@@ -3,114 +3,188 @@ layout: "newrelic"
...
 
-## Argument Reference
+```hcl
+resource "newrelic_synthetics_private_location" "bar1" {
+  description               = "Test Description-Updated"
+  name                      = "%[1]S"
+  verified_script_execution = true
+}
+resource "newrelic_synthetics_monitor" "foo" {
+  custom_header{
```
Comment (sanderblue) [inline website/docs/r/synthetics_monitor.html.markdown:50]: Not a blocker, but we should get your set up with a Terraform HCL formatter for your IDE so we have consistent code style for our examples.

File: website/docs/r/synthetics_script_monitor.html.markdown [hunk 12]
```diff
@@ -0,0 +1,156 @@
...
+-> **NOTE:** The [newrelic_synthetics_private_location](newrelic_synthetics_private_location.html) resource private minion can take upto 10 minutes to be available through Terraform.
+
+## Example Usage
+
+##### Type: `SCRIPT_API`
+
+-> **NOTE:** The preferred runtime is `NODE_16.10.0` while configuring the `SCRIPT_API` monitor. Other runtime may be deprecated in the future and receive fewer product updates. 
+
+```hcl
+    resource "newrelic_synthetics_script_monitor" "foo" {
```
Comment (sanderblue) [inline website/docs/r/synthetics_script_monitor.html.markdown:22]: Similar issue regarding the HCL format here

**Pull request merged**